Aquality.Selenium.Template/Aquality.Selenium.Template.NUnit/Tests/BaseTest.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Aquality.Selenium.Browsers;
+using Aquality.Selenium.Core.Logging;
+using Aquality.Selenium.Template.Implementation;
```

Comment:
This namespace name "Implementation" does not sound right. Could you please choose more specific name?

Reply:
This namespace was renamed to CustomConfigImplementation